src/core/server/rendering/views/styles.tsx

```diff
@@ -159,6 +159,16 @@ export const Styles: FunctionComponent<Props> = ({ darkMode }) => {
             background-color: ${darkMode ? '#1BA9F5' : '#006DE4'};
           }
 
+          .loadingLogoContainer {
```

Comment:
why height is 60px? thx

Reply:
+1. Should this container be a fixed size?

Reply:
I chose to only restrict height but not width so we can resize the logo proportionally while preserving the width to height ratio of the original logo. 

Currently I did not receive a strict size requirement on the logos, so I restrict height to 60 px for mimic the look of the original opensearch loading logo on the loading page.

Reply:
I see. Thanks for the explain. One more comment below.